src/vs/platform/terminal/common/terminal.ts

```diff
@@ -613,4 +613,20 @@ export interface ITerminalProfileSource extends IBaseUnresolvedTerminalProfile {
 	source: ProfileSource;
 }
 
-export type ITerminalProfileObject = ITerminalExecutable | ITerminalProfileSource | null;
+export interface ITerminalContributions {
+	profiles?: ITerminalProfileContribution[];
+}
```

Comment:
This is probably better moved to terminalExtensionPoints.ts as it's not touched in platform/ and only touched in that file.

Reply:
has to be there bc of cross platform violations